### DISCUSSION TITLE
Kick Reconnect Loop

### DISCUSSION THREAD

**Discussion by ghost** [sorayuki/obs-multi-rtmp#434] [Q&A]
So every time I try to stream to kick, in about 6 seconds, it reconnects, and then it loops, I have tried every video encoder and nothing, I have also reset my stream key too. I want to stream to kick so I can build up more viewers. Is there any way to fix the problem and has anybody else had this issue.

**Comment by elxalejo** [comment]
If the same thing happened to me as to your colleague, he disconnects and reconnects again and is like that all the time.
